packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -430,3 +430,34 @@ prometheus.io/scrape: "true"
 prometheus.io/path:  "{{ $pprof.path | default "/debug/metrics/prometheus" }}"
 prometheus.io/port: "{{ $pprof.port | default 6060 }}"
 {{- end -}}
+
+{{- /* Needs a serviceAccountName in the pod with permissions to access gstorage */ -}}
+{{- define "common.gsutil-sync-data-init-container" -}}
+- name: gsutil-sync-data
+  image: gcr.io/google.com/cloudsdktool/cloud-sdk:latest
+  imagePullPolicy: Always
+  command:
+  - /bin/sh
+  - -c
+  args:
+  - |
+     # If older than upload period, remove the chain data dir.
+     if [ -d /root/.celo/celo/chaindata ]; then
+       mtime=$(stat -f "%Y" /root/.celo/celo/chaindata)
```

Comment:
Was trying to see how this command works and I can't seem to get this to work for me:
```
$ docker run --rm -it gcr.io/google.com/cloudsdktool/cloud-sdk:latest sh
# mkdir yo
# stat -f "%Y" yo
stat: cannot read file system information for '%Y': No such file or directory
  File: "yo"
    ID: e0ec7fa141d2f5aa Namelen: 255     Type: overlayfs
Block size: 4096       Fundamental block size: 4096
Blocks: Total: 15313873   Free: 10762481   Available: 9977150
Inodes: Total: 3907584    Free: 2947913
```

Reply:
Yea, my bad, I got it working in locally the container (running it locally is actually super helpful, I didn't know how to do that before)